masonry/src/passes/update.rs

```diff
@@ -677,6 +686,21 @@ pub(crate) fn run_update_pointer_pass(root: &mut RenderRoot) {
         }
     }
 
+    if prev_hovered_widget != next_hovered_widget {
+        if let Some(prev_hovered_widget) = prev_hovered_widget {
+            run_single_update_pass(root, prev_hovered_widget, |widget, ctx| {
```

Comment:
This panics if `prev_hovered_widget` is no longer in the tree...

Reply:
Fixed in #834.